.circleci/config.yml

```diff
@@ -20,9 +20,22 @@ jobs:
       - run:
           name: Run Unit Tests
           command: go test -v ./... -short
+      - run:
+          name: Install AWS CLI
+          command: |
+            sudo apt-get -y -qq update
+            sudo apt-get -y -qq install awscli
```

Comment:
this is an old version of the awscli that will prob bite you down the road. The preferred way to install it is in the `deploy_to_dev` job below